glusterd2/commands/volumes/volume-delete.go

```diff
@@ -86,6 +87,12 @@ func volumeDeleteHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if len(volinfo.SnapList) > 0 {
+		errMsg := fmt.Sprintf("Cannot delete Volume %s ,as it has %d snapshots. To delete the volume, first delete all the snapshots under it..", volname, len(volinfo.SnapList))
```

Comment:
IMO, this error message is too lengthy for the user, can we simplify it?

Reply:
Done